packages/api/src/feedback.ts

```diff
@@ -16,7 +16,9 @@
  * SPDX-License-Identifier: Apache-2.0
  ***********************************************************************/
 
-export type FeedbackCategory = 'developers' | 'feature' | 'bug';
+export type FeedbackCategory = 'developers' | 'design' | 'feature' | 'bug';
+
+export type DirectFeedbackCategory = 'developers' | 'design';
```

Comment:
To avoid duplicating keys, you may use:


```suggestion
export type DirectFeedbackCategory = 'developers' | 'design';

export type FeedbackCategory = DirectFeedbackCategory | 'feature' | 'bug';

```

Reply:
Nice, thanks for suggestion!